src/UnitTests/Analysis/DataFlowAnalysisTests2.cs

```diff
@@ -229,7 +229,8 @@ void test(int32 a, int32 b)
 test_entry:
 	// succ:  l1
 l1:
-	Mem8[0x00010008:word32] = a + b
+	word32 r1_7 = a + b
+	Mem8[0x00010008:word32] = r1_7
```

Comment:
This PR contains `SsaTransform` adding use instruction changes. It marks identifiers of exit block use statements as `used`. See 531 line of `SsaTransform` (https://github.com/uxmal/reko/pull/385/commits/85de2c412b2bdc7b1fad7e06a46e6ac8c6bbcc1a#diff-c32fd9c08cbe70e692e0b2a49b8810fa)
So, now there are two uses of `r1_7` (`word32 r1_7 = a + b` and `use r1_7`) and it could not be coalesced.
I see, it does not respect user signature but it's another issue. `DataFlowAnalysis2Tests` is early version of new `ssa`. So it should be solved at `analysis-development` (if it is not already fixed)